docs/advanced/worker-pools.md

```diff
@@ -1,6 +1,6 @@
 # Worker Pools
 
-Lokomotive on AWS, Azure, Google Cloud and Packet allows additional groups of workers to be defined and joined to a cluster. For example, add worker pools of instances with different types, disk sizes, Container Linux channels, or preemptibility modes.
+Lokomotive on AWS, Azure, Google Cloud and Packet allows additional groups of workers to be defined and joined to a cluster. For example, add worker pools of instances with different types, disk sizes, Flatcar Container Linux channels, or preemptibility modes.
 
```

Comment:
Shouldn't it be `CoreOS Container Linux`?

Reply:
I'm not sure, I think this documentation is exclusively for Flatcar Container Linux, so not reason to talk about CoreOS Container Linux in this doc.

Reply:
ok, that makes sense, if you say so.